deploy/docker-compose/README.md

```diff
@@ -0,0 +1,261 @@
+# MONAI Deploy Lite
+
+MONAI Deploy Lite provides a simple & lightweight solution for running MONAI Applications (MAPs) on a minimal MONAI Deploy platform.
+
+It is meant to bridge the big gap between the local development of MAPs in a laptop using the MONAI Deploy App SDK and a production-ready deployment. 
+
+Users of MONAI Deploy Lite will be able to run their MAPs, connected to a test PACS, or their own test/research PACS, for further validation, confidently taking steps towards production.   
+
+Reusing the same essential core services for DICOM I/O and AI workflow orchestration provides the same functionality and consistent experience independently of where and how the applications are run, with minimal changes for the end user.
+
+
+## Prerequisites
+
+- [Docker Engine](https://docs.docker.com/engine/install/) v20.10.18+
+- [Docker Compose Plug-in](https://docs.docker.com/compose/install/) v2.10.2+
+- MONAI Deploy MAPs built using the [MONAI Deploy App SDK](https://github.com/Project-MONAI/monai-deploy-app-sdk)
+- [jq](https://stedolan.github.io/jq/)
+- (Optional) [Informatics Gateway CLI](https://github.com/Project-MONAI/monai-deploy-informatics-gateway/releases)
+
+## Installation
+
+To use MONAI Deploy Lite, install all prerequisites & download this entire directory from [GitHub](https://github.com/Project-MONAI/monai-deploy/tree/main/docker-compose/deploy/docker-compos). 
+
+## Start the Services
+
+The [docker compose file](docker-compose.yml) spins up the following services. Each service may be accessed at the IP and port described below using the default [.env](.env) configuration file.
+
+Execute the following command to bring up all services.
+
+```bash
+docker compose up
+docker compose up -d # or run detached
+docker compose logs -t -f # view output from all containers
+```
```

Comment:
do we run all three commands? Or just one of those?

Reply:
any of those, depends on ur needs